packages/amplication-prisma-db/prisma/migrations/20220707120437_from_app_to_resource/migration.sql

```diff
@@ -63,6 +63,6 @@ ALTER INDEX "EntityPermissionRole.entityVersionId_action_appRoleId_unique" RENAM
 CREATE TYPE "EnumResourceType" AS ENUM ('Service');
 
 -- Add value to current resources
-ALTER TABLE "Resource" ADD COLUMN     "type" "EnumResourceType" NULL;
-UPDATE "Resource" SET type = 'Service';
-ALTER TABLE "Resource" ALTER COLUMN "type" SET NOT NULL;
```

Comment:
@tupe12334 we should not alter migrations manually unless there is a real custom changes we need to implement and there is no other way.
We should never alter migrations after they are committed - by doing it you are breaking the environment of anyone who already pulled the change. 

Is there a reason why you could not create a new migration? 

@arielweinberger this should be included in our best practices and be circled around to everyone on the team